plugins/com.google.cloud.tools.eclipse.login/src/com/google/cloud/tools/eclipse/login/ui/AccountsPanel.java

```diff
@@ -29,25 +30,24 @@
 import org.eclipse.swt.events.SelectionAdapter;
 import org.eclipse.swt.events.SelectionEvent;
 import org.eclipse.swt.graphics.Color;
+import org.eclipse.swt.layout.GridData;
 import org.eclipse.swt.widgets.Button;
 import org.eclipse.swt.widgets.Composite;
 import org.eclipse.swt.widgets.Control;
 import org.eclipse.swt.widgets.Label;
 import org.eclipse.swt.widgets.Shell;
 
-import java.util.ArrayList;
-import java.util.List;
-
 /**
  * A panel listing all currently logged-in accounts. The panel allows adding new accounts and
  * logging out all accounts.
  */
 public class AccountsPanel extends PopupDialog {
 
-  private IGoogleLoginService loginService;
+  private final IGoogleLoginService loginService;
 
   @VisibleForTesting Button logOutButton;
-  @VisibleForTesting List<Label> accountLabels = new ArrayList<>();
+  @VisibleForTesting final List<Label> nameLabels = new ArrayList<>();
```

Comment:
Tried to avoid setting an id on the labels, since we are not solving the problem of having code only for testing, but I don't really find a good alternative. I considered different things, but they have downsides.